extension/otto/otto_test.go

```diff
@@ -1859,6 +1859,61 @@ var _ = Describe("Otto extension manager", func() {
 			Expect(context).To(HaveKeyWithValue("resp", HaveKeyWithValue("data", map[string]interface{}{})))
 		})
 	})
+
+	Describe("Using gohan_sync_delete builtin", func() {
+		It("Should delete sync", func() {
+			extension, err := schema.NewExtension(map[string]interface{}{
+				"id": "test_extension",
+				"code": `
+					gohan_register_handler(
+						"test_event",
+					 	function(context) {
+							gohan_sync_delete("/gohan_sync_delete_test");
```

Comment:
2nd parameter is optional. Default is false. Here, I tested single argument case.